bpf/bpf_netdev.c

```diff
@@ -120,10 +120,12 @@ static inline int handle_ipv6(struct __sk_buff *skb, __u32 src_identity)
 		if (ret < 0)
 			return ret;
 	}
-#if defined(ENCAP_IFINDEX) || defined(NO_REDIRECT)
+# ifdef ENABLE_IPT
+#  if defined(ENCAP_IFINDEX) || defined(NO_REDIRECT)
```

Comment:
Nit: you can probably replace `NO_REDIRECT` with `ENABLE_IPT`.

Reply:
Yeah, actually I'll get rid of ENABLE_IPT and just define the NO_REDIRECT in node_config.h instead.